source/_docs/https-load-balancer.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Supported Methods for Configuring an HTTPS Load Balancer
+description: Explore different solutions for enabling HTTPS on all environment URLs (including Multidev) associated with your organization.
+categories: [developing]
+tags: [domains, organizations]
+---
+
+## Shared Load Balancer and CloudFlare's Universal SSL (Free)  
+Use CloudFlare's free [Universal SSL](https://blog.cloudflare.com/introducing-universal-ssl/) option with Pantheon's shared load balancer to enable HTTPS across all site URLs that are associated with your organization for free.
+
+### Configuration
+1. [Request a Vanity domain](/docs/vanity-domains/#request-the-vanity-domain) for your organization and [consider the effects](/docs/vanity-domains/#effects-and-considerations) before proceeding.
```

Comment:
These instructions do not belong here. Vanity domains are a similar thing for an agency, but not really in scope for this doc.